com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -1326,6 +1327,25 @@ internal void HandleIncomingData(ulong clientId, NetworkChannel networkChannel,
                                 }
                             }
 
+                            break;
+                        }
+                    case NetworkConstants.PARENT_SYNC:
+                        {
+                            if (IsClient)
+                            {
+                                using (var reader = PooledNetworkReader.Get(messageStream))
+                                {
+                                    var networkObjectId = reader.ReadUInt64Packed();
+                                    var (isReparented, latestParent) = NetworkObject.ReadNetworkParenting(reader);
+                                    if (SpawnManager.SpawnedObjects.ContainsKey(networkObjectId))
```

Comment:
I would have expected some kind of message or other handling for the case where the spawn manager doesn't know about a network object id that was transmitted

Reply:
will `NetworkLog` a `Warning` message here.

Reply:
done.